src/ochubryk/Hometask2/price.py

```diff
@@ -0,0 +1,36 @@
+# Напишите программу, которая считает общую цену.
+M_rubles = int(input('Введите рубли: '))
+N_pennies = int(input('Введите копейки: '))
+L_count = int(input('Введите количество товара: '))
+
+All_N_pennies = N_pennies * L_count
+if All_N_pennies > 99:
```

Comment:
it can be made easier if we'll first transform all to pennies and next find rubles and penny accordingly

Reply:
Fixed